docs/content/direct/get-started.md

```diff
@@ -9,11 +9,29 @@ This page shows one concrete example of steps 2--7 from the [full Installation a
     1. Create and register two WECs.
   1. Exercise KubeStellar
 
+
+
 ## Setup
 
 This is one way to produce a very simple system, suitable for study but not production usage. For general setup information, see [the full story](user-guide-intro.md#the-full-story).
 
-### Install software prerequisites
+### Automated Shell Script
+
+There are two ways to run the setup: using our automated shell script or running the commands step-by-step. For a quick and easy setup of a KubeStellar demo environment, you can use the provided script. This script automates the process of creating a KubeStellar environment suitable for demonstration purposes. It performs the following tasks:
```

Comment:
> For a quick and easy setup of a KubeStellar demo environment.

I think that this does not set quite the right expectation. The wording implies, to me, a false dichotomy --- between "install KubeStellar" as exemplified by the script and "use KubeStellar". KubeStellar is designed to allow more than one ITS, more than one WDS, and allow them to be introduced and retired over time. One could reasonably consider this to be part of "using" KubeStellar. See https://docs.kubestellar.io/release-0.24.0/direct/user-guide-intro/#the-full-story , which dances around drawing a particular line between "install" and "use". In different usage scenarios that line could be drawn differently, or not at all. To be clear, I am not objecting to having a script that wraps up some stuff, my concern is how it is described. The existing Getting Started steps teach about the structure involved.